### ISSUE TITLE
Keep variables after compilation

### ISSUE THREAD

**Issue by kevin-bog** [sagold/handlebars-webpack-plugin#59]
Hi, firstly, thanks for your work !

Everything works fine except, I would like to keep my variable ```{{}}``` after compilation. Because this is my controller which send data to my template how can i do this ?

Edit;
Look like I couldn't do this. I had to handle this from the server and build one entry by partial ?

Thanks for taking the time

**Comment by sagold** [comment]
Hi. 

Can you rephrase your question? What do you mean by "keep the variable {{}}"

- is it the string within the hbs-template
- or something wirh the input-data

And besides: Is this a question about handlebars or this plugin? If you have the an example of your input and expected output, i might be able to help.

**Comment by kevin-bog** [comment]
Hey,

sorry I meant,

I want to keep the handle bars variables : 
```<p>{{ firstname }}</p>```
after the compilation, because after compilation, my variables are removed:
```<p></p>```

Look like I made a mistake, and had to handle the rendering all from my serverside, without compiling the file

**Comment by lamualfa** [comment]
For me, just add backslash before your variable. Example:

Before compilation:
```hbs
<p>\{{ firstname }}</p>
```
After compilation:

```hbs
<p>{{ firstname }}</p>
```

**Comment by xavierfoucrier** [comment]
See https://handlebarsjs.com/guide/expressions.html#html-escaping.
